src/components/v1/V1Project/ProjectActivity/index.tsx

```diff
@@ -181,36 +173,25 @@ export default function ProjectActivity() {
         group.map((e: ProjectEvent) => {
           let elem: JSX.Element | undefined = undefined
 
-          if (e.payEvent) {
-            elem = <PayEventElem event={e.payEvent as PayEvent} />
+          if (
+            e.payEvent ||
+            e.redeemEvent ||
+            e.projectCreateEvent ||
+            e.deployedERC20Event
+          ) {
+            elem = <ActivityEvent event={e} />
```

Comment:
LOOOOVE

wonder if there's a good way to move this `if (a || b || c)` logic to the ActivityEvent, so ProjectActivity doesn't have to care about which events have elements?

i guess if ActivityEvent renders `null`, then elem will == null?